mobile/pubspec.yaml

```diff
@@ -25,19 +25,23 @@ dependencies:
     sdk: flutter
 
   # Adds support for iOS icons
-  cupertino_icons: ^0.1.2
+  cupertino_icons: 0.1.3
   # Firebase dependencies
-  cloud_firestore: ^0.8.2
-  firebase_auth: ^0.6.6
-  google_sign_in: ^3.2.4
+  cloud_firestore: ^0.13.5
+  firebase_auth: ^0.16.0
+  google_sign_in: ^4.4.4
   # Device scanner dependencies
-  camera: ^0.2.9
-  firebase_ml_vision: ^0.2.1
+  camera: ^0.5.7+4
+  firebase_ml_vision: ^0.9.3+8
+
 
 dev_dependencies:
   flutter_test:
     sdk: flutter
 
+module:
+    androidX: true // for migrating for androidx
+
```

Comment:
This block should not be needed since since the project contains an `android/` directory and it not treated as an Android module: https://flutter.dev/docs/development/androidx-migration

I was able to get your changes to build without this included.